src/coreclr/nativeaot/System.Private.CoreLib/src/System/Runtime/InteropServices/ComWrappers.CoreRT.cs

```diff
@@ -240,6 +365,11 @@ public unsafe IntPtr GetOrCreateComInterfaceForObject(object instance, CreateCom
                 runtimeDefinedVtable[runtimeDefinedCount++] = DefaultIUnknownVftblPtr;
             }
 
+            if ((flags & CreateComInterfaceFlags.TrackerSupport) == CreateComInterfaceFlags.TrackerSupport)
```

Comment:
```suggestion
            if ((flags & CreateComInterfaceFlags.TrackerSupport) != 0)
```
Shorted and less duplication